tests/files/tests.py

```diff
@@ -340,6 +341,37 @@ def test_file_move_overwrite(self):
         os.close(handle_a)
         os.close(handle_b)
 
+    def test_file_move_copystat_cifs(self):
+        handle_a, self.file_a = tempfile.mkstemp()
+        handle_b, self.file_b = tempfile.mkstemp()
+
```

Comment:
@timgraham wilco. As new contributors, it's difficult to know which direction so I appreciate the explanation.